src/components/Header/components/MenuWrapper/index.tsx

```diff
@@ -1,13 +1,18 @@
-import React, { FC } from 'react';
+import React, { FC, useState } from 'react';
 import { CoursesSelect } from './components/CoursesSelect';
 import { Nav } from './components/Nav';
 import { StyledMenuWrapper } from './styled';
 
 export const MenuWrapper: FC = () => {
+  const [displayCoursesList, setDisplayCoursesList] = useState<boolean>(false);
+
   return (
     <StyledMenuWrapper>
-      <Nav />
-      <CoursesSelect />
+      <Nav setDisplayCoursesList={setDisplayCoursesList} />
+      <CoursesSelect
+        displayCoursesList={displayCoursesList}
```

Comment:
you can also write 
<CoursesSelect {...{displayCoursesList, setDisplayCoursesList}} />